includes/class-wpcom-legacy-redirector.php

```diff
@@ -137,7 +137,7 @@ public static function wpcom_legacy_add_redirect_js( $hook ) {
 		}
 
 		wp_enqueue_script( 'wpcom-legacy-redirector', plugins_url( '/../js/admin-add-redirects.js', __FILE__ ), [], WPCOM_LEGACY_REDIRECTOR_VERSION, true );
-		wp_localize_script( 'wpcom-legacy-redirector', 'wpcomLegacyRedirector', array( 'siteurl' => get_option( 'siteurl' ) ) );
+		wp_localize_script( 'wpcom-legacy-redirector', 'wpcomLegacyRedirector', array( 'siteurl' => home_url() ) );
```

Comment:
This change seems unrelated, but might be good to pull into a new issue + PR to explain why this change was warranted.